Realm/Realm.Sync/Permissions/ManagementObjectStatus.cs

```diff
@@ -16,26 +16,29 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+using System;
+
 namespace Realms.Sync
 {
     /// <summary>
     /// The status of the management object as set by the server.
     /// </summary>
+    [Flags]
```

Comment:
Ah, that does make sense. Interesting situation actually -- proves that there are times when you want to be able to combine mutually exclusive flags. I am ok with keeping it like it is for this.